src/sentry/static/sentry/app/components/dropdownControl.tsx

```diff
@@ -1,51 +1,68 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import styled from '@emotion/styled';
 
 import DropdownBubble from 'app/components/dropdownBubble';
-import DropdownButton from 'app/components/dropdownButton';
-import DropdownMenu from 'app/components/dropdownMenu';
+import DropdownButton, {
+  Props as DropdownButtonProps,
+} from 'app/components/dropdownButton';
+import DropdownMenu, {GetActorPropsFn} from 'app/components/dropdownMenu';
 import MenuItem from 'app/components/menuItem';
+import theme from 'app/utils/theme';
+
+type DefaultProps = {
+  /**
+   * Should the menu contents always be rendered?  Defaults to true.
+   * Set to false to have menu contents removed from the DOM on close.
+   */
+  alwaysRenderMenu: boolean;
+  /**
+   * Width of the menu. Defaults to 100% of the button width.
+   */
+  menuWidth: string;
+};
+
+type Props = DefaultProps & {
+  /**
+   * String or element for the button contents.
+   */
+  label?: string | React.ReactNode | null;
```

Comment:
`ReactNode` includes string and null type, so you don't need to put them here.

Here's the relevant bit in React's `index.d.ts`
```
type ReactText = string | number;
type ReactChild = ReactElement | ReactText;

interface ReactNodeArray extends Array<ReactNode> {}
type ReactFragment = {} | ReactNodeArray;
type ReactNode = ReactChild | ReactFragment | ReactPortal | boolean | null | undefined;
```

Reply:
Thanks @leedongwei , cleaning that up makes sense